Applications/WebPortal/WebApp/library/source/pages/account_page/profile_page/photo_field.tsx

```diff
@@ -48,6 +64,10 @@ export class PhotoField extends React.Component<Properties, {}> {
     onSubmit: () => {}
   };
 
+  constructor(properties: Properties) {
```

Comment:
You can remove the constructor in this case where all it does is call the parent's constructor.